src/elements/Icon/Icon.js

```diff
@@ -1,13 +1,13 @@
 import cx from 'classnames'
 import React, { PropTypes } from 'react'
 
-import META from '../../utils/Meta'
-import * as sui from '../../utils/semanticUtils'
 import {
+  getUnhandledProps,
+  META,
+  SUI,
   useKeyOnly,
   useValueAndKey,
-  getUnhandledProps,
-} from '../../utils/propUtils'
+} from '../../lib'
```

Comment:
Good example of the improvement in organization, this used to require importing from 3 different locations.